### ISSUE TITLE
AA issue: Can't connect to headunit process

### ISSUE THREAD

**Issue by MarioAM3** [gartnera/headunit#181]
"_startContextReady: Can't connect to headunit process" issue everytime I tried to connect my smartphone. I can see the same issue when I turn on the CMU without a smartphone connected.
In AIO 2.8.4 never happend before.

FW Version: 56.00.230
AIO Version: 2.8.5
Smartphone: Samsung Galaxy S7 (Android 8.0)

**Comment by Trevelopment** [comment]
If you are getting the "cant connect to headunit" message before you even connect your phone that means you need to reinstall AA in your car.

**Comment by MarioAM3** [comment]
Hi, a clean installation or reinstall over the previous AA?.

**Comment by Trevelopment** [comment]
Whichever one works .  You should be able to install over it but if that doesn't work try clean.

**Comment by MarioAM3** [comment]
> 
> 
> Whichever one works . You should be able to install over it but if that doesn't work try clean.

Hi, I installed AA 1.12 (AIO 2.8.5) over it until three times without success. 
After that, I tried doing a fresh installation, same results. In both cases the "can´t connect" issue still appears without any smartphone connected. I think something is wrong with new AA or AIO.
I found a forum with people in my same situation. 
https://www.mazda3revolution.com/threads/android-auto-wont-launch.239247/
